### ISSUE TITLE
Report a reason into a Pull Request when kodiak closes it

### ISSUE THREAD

**Issue by clxmstaab** [chdsbd/kodiak#249]
Feature Request:

'sometimes' we see kodiak closing pull requests... we dont know why.. would be great kodiak would report a comment to tell us why it made this decision.

Maybe-Repro:

I guess it happens in the following situation.

follow branch structure
```
- master (1)
  |- feature1 (2)
     |- another PR based on feature1 (3)
````

it seems kodiak is closing branch (3) when branch (2) gets merged (via kodiak automerge-label).

**Comment by chdsbd** [comment]
I think this should be fixed in https://github.com/chdsbd/kodiak/pull/242

**Comment by chdsbd** [comment]
This was fixed with #242, which was deployed in [v0.15.0](https://github.com/chdsbd/kodiak/releases/tag/v0.15.0).
